reframe/utility/__init__.py

```diff
@@ -727,6 +727,22 @@ def update(self, other):
             for k, v in scope_dict.items():
                 self.data[scope][k] = v
 
+    def scope(self, scope):
```

Comment:
You can mention this on line 673 where you comment on trying to retrieve whole scopes.